web/services/AI/ai_model.py

```
@@ -0,0 +1,173 @@
+import json
+import os
+import re
+from typing import Union
+
+import google.generativeai as genai
+
+api_key = os.getenv("GOOGLE_API_KEY")
+
+if not api_key:
+    raise ValueError("GOOGLE_API_KEY not found in environment variables")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)